index.bs

```diff
@@ -2775,45 +4881,120 @@ partial interface MLGraphBuilder {
     tensor is specified by the *newShape* argument.
 </div>
 
-### The sigmoid() method ### {#api-mlgraphbuilder-sigmoid}
+<details open>
+  <summary>
+    The {{MLGraphBuilder/reshape(input, newShape)}} steps are:
+  </summary>
+  <div algorithm=reshape class=algorithm-steps>
+    1. If |input| is not an instance of {{MLOperand}}, then throw a "{{TypeError}}" {{DOMException}} and stop.
+    1. Let |outputShape| be an empty array of {{unsigned long}}.
+    1. If |newShape| is a scalar [=number=], set |outputShape| to `[ 1 ]`.
+    1. Otherwise, if |newShape| is an array of {{unsigned long}}:
+        1. If the size of |newShape| is `0`, set |outputShape| to `[ 1 ]` (reshaping to scalar).
+        1. If |newShape| contains more than one `null` value, then throw a "{{DataError}}" {{DOMException}} and stop.
+        1. If any value in |newShape| is `0`, then throw a "{{DataError}}" {{DOMException}} and stop.
```

Comment:
There's a relevant discussion https://github.com/webmachinelearning/webnn/issues/391 - Allow 0 size dimensions .

Reply:
Thanks for linking, it will keep reference to this dependency, when that issue is to be closed. 
At this point we don't need to modify this step yet.